src/Carbon/Carbon.php

```diff
@@ -396,11 +396,9 @@ public function __construct($time = null, $tz = null)
             )
         ) {
             $dateTime = new DateTime('now', $timezone);
-            $microTime = microtime(true) * 1000000 % 1000000;
-            if ($microTime > 0) {
-                $microTime = str_pad(strval($microTime), 6, '0', STR_PAD_LEFT);
-                $time = $dateTime->format(static::DEFAULT_TO_STRING_FORMAT).'.'.$microTime;
-            }
+            $microTime = microtime(true);
```

Comment:
Can we just replace this 2 lines with: `$microTime = substr(microtime(false), 2, 6)` ?